### ISSUE TITLE
macOS: voiceover does not read ARIA alert anymore

### ISSUE THREAD

**Issue by Benjin** [microsoft/vscode#80017]
**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.9.0 (user setup)
Commit: 78a42e1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAC 10.14.6
MAS 1.3.1
Tools : Voice over 

Voiceover doesn't read toast details when popped up.  Possibly upstream to Electron?

![image](https://user-images.githubusercontent.com/1609827/63894601-df03da00-c9a1-11e9-9fca-95a8ae940a8d.png)

**Comment by weinand** [comment]
Please file issue against SQL extension.

**Comment by Benjin** [comment]
The toast is being produced by our (Azure Data Studio) call to `vscode.window.showWarningMessage()`, this should point you in the right direction.

**Comment by isidorn** [comment]
Assigning first to @bpasero since he owns the notifications
Though it makes sense to me that the Notification is read out as an "alert" since it is exactly that at the end of the day.

**Comment by bpasero** [comment]
![image](https://user-images.githubusercontent.com/900690/64596723-b70a7280-d3b4-11e9-9c53-02d551998bd9.png)

Sounds upstream to me, notification toasts have been calling into `aria.alert()` since day 1. 

@isidorn is there a known issue already that macOS voice over does not read out our `aria.alert()`?

**Comment by isidorn** [comment]
@bpasero no, there is no known issue. What might be happening here is that the alert gets read out quickly and then something else gets read out becuase the focus automatically moves?
Try doing the alert with some timeout just as an experiment to test my theory.

**Comment by weinand** [comment]
From the screenshot it looks like that there is a WebView involved...

**Comment by bpasero** [comment]
This is broken for me everywhere, unrelated to notifications, e.g. also explorer:

![image](https://user-images.githubusercontent.com/900690/64613590-c995a380-d3d6-11e9-95fb-a618f50d2f73.png)

**Comment by isidorn** [comment]
This works fine for me with latest out of source. Maybe the latest electron fixed it, thus closing.
@Benjin if you still see this with latest vscode insiders let us know and we can reopen

![Screenshot 2019-10-25 at 12 42 20](https://user-images.githubusercontent.com/1926584/67565111-ed157300-f724-11e9-876f-119e20fe81c5.png)
